Agents/FeatureInfoAgent/sample/fia/trajectory_meta.sparql

```diff
@@ -0,0 +1,12 @@
+PREFIX disp: <https://www.theworldavatar.com/kg/ontodispersion/>
+
+SELECT ?Property (AVG(?value) AS ?Value) ?Unit
+WHERE {
+  SERVICE [ONTOP] {
+    ?Feature disp:hasValue ?value.
+  }
+  BIND("Average NOx concentration" AS ?Property)
+  BIND("&micro;g/m&sup3;" AS ?Unit)
+}
+GROUP BY ?Property ?Unit
+
```

Comment:
Not overly relevant to this feature but it should be noted that this isn't a correct calculation of the average concentration.

Reply:
I am aware it is not accurate as it does not take into account the duration spent in each polygon